config/jobs/kubernetes-sigs/azurefile-csi-driver/azurefile-csi-driver-config.yaml

```diff
@@ -605,8 +605,8 @@ presubmits:
               value: "true"
             - name: TEST_WINDOWS
               value: "true"
-            - name: WINDOWS_FLAVOR
```

Comment:
We should drop this since the job name is `pull-azurefile-csi-driver-e2e-capz-windows-2019`

Reply:
will this do the trick?

```
            - name: WINDOWS_SERVER_VERSION
               value: "windows-2019"
```

Reply:
it should yes.

Reply:
just remove following config since it's running on 2019 actually
```
            - name: WINDOWS_FLAVOR
              value: "containerd-2022"
```

Reply:
Done